src/types/modal/addModal.type.ts

```diff
@@ -0,0 +1,4 @@
+export type AddModalType = {
```

Comment:
Props can be specified directly above the component, there is no need to move one type to a separate folder and file